modules/gapi/test/cpu/gapi_imgproc_tests_cpu.cpp

```diff
@@ -176,6 +176,27 @@ INSTANTIATE_TEST_CASE_P(SobelXYTestCPU32F, SobelXYTest,
                                 Values(BORDER_CONSTANT, BORDER_REPLICATE, BORDER_REFLECT),
                                 Values(0, 1, 255)));
 
+INSTANTIATE_TEST_CASE_P(LaplacianTestCPU, LaplacianTest,
+                        Combine(Values(CV_8UC1, CV_8UC3, CV_16SC1),
+                                Values(cv::Size(1280, 720),
+                                       cv::Size(640, 480)),
+                                Values(-1, CV_16S, CV_32F),
+                                Values(IMGPROC_CPU),
+                                Values(AbsExact().to_compare_obj()),
+                                Values(3, 5)));
```

Comment:
Are you sure you shouldn't check more parameters of Laplacian? You can see what guys in imgproc_tests and perf_tests check if you haven't yet. If you sure it's enough then OK

Reply:
Mb it's not needed in gapi_perf_tests, but in accuracy tests we must become confident that the kernel works quite similarly to how OpenCV function does with (almost) all range of parameters. Especially for GPU implementation.

Check for bilateral as well, please

Reply:
Extended.